zulipterminal/model.py

```diff
@@ -468,9 +469,8 @@ def update_stream_message(self, topic: str, message_id: int,
 
         return response['result'] == 'success'
 
-    def generate_all_emoji_data(self,
-                                custom_emoji: Dict[str,
-                                                   Any]) -> NamedEmojiData:
+    def generate_all_emoji_data(self, custom_emoji:
+                                Dict[str, RealmEmojiData]) -> NamedEmojiData:
```

Comment:
```
    def generate_all_emoji_data(self, custom_emoji: Dict[str,
                                RealmEmojiData]) -> NamedEmojiData:
```
Is this how it should be because otherwise character limit gets exceeded.

Reply:
See the comment in the next review.